buddy_mentorship/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.utils import timezone
+from apps.users.models import User
+
+
+class BuddyRequest(models.Model):
+    request_sent = models.DateTimeField(default=timezone.now)
+    requestee = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="requestee"
+    )
+    requestor = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="requestor"
+    )
+    message = models.CharField(max_length=511)
```

Comment:
should we make this field a TextField type so that a user may write a long message?

Reply:
@rifferreinert Comment addressed - made the message a TextField.